Header header
uint8 missionEndAction  #1 for Hover, 2 for RTH, 3 for land
uint8 uavSpeed #
